File: src/benchmarks/network/http_server/golang.default/httpserver.go
package main

import (
    "fmt"
    "net/http"
)

func test(w http.ResponseWriter, req *http.Request) {
	w.Header().Set("Content-Type", "text/plain; charset=UTF-8")
	w.Header().Set("Server", "Example")
	w.Header().Set("Date", "Wed, 17 Apr 2013 12:00:00 GMT")
    fmt.Fprintf(w, "Hello, World!\r\n")
}

func main() {
    http.HandleFunc("/plaintext", test)
    http.ListenAndServe(":8080", nil)
}

File: src/benchmarks/network/http_server/cpp-httplib/httpserver.cpp
#include "./cpp-httplib/httplib.h"

using namespace httplib;

int main(void) {
  Server svr;

  svr.Get("/plaintext", [](const Request & /*req*/, Response &res) {
      res.set_header("Server", "Example");
      res.set_header("Date", "Wed, 17 Apr 2013 12:00:00 GMT");
      res.set_header("Content-Type", "text/plain; charset=UTF-8");
      res.set_content("Hello World!\r\n", "text/plain");
  });

  svr.listen("localhost", 8080);
}